ethers-providers/src/transports/quorum.rs

```diff
@@ -190,15 +191,67 @@ impl<T: JsonRpcClientWrapper> QuorumProvider<T> {
 
         let mut numbers = vec![];
         let mut errors = vec![];
+
+        // exit once required quorum weight is achieved
```

Comment:
nit: this isn't quite what we do right? possibly misleading for future readers - maybe this is a good spot to describe the general strategy

Reply:
okay :+1: